cmd/skaffold/app/cmd/dev.go

```diff
@@ -55,6 +55,9 @@ func runDev(ctx context.Context, out io.Writer) error {
 		}()
 	}
 
+	// force skaffold labels to be applied in dev mode - port forwarding and log tailing won't work without them
+	opts.AddSkaffoldLabels = true
+
```

Comment:
I think it's better to set this with the `DefValuePerCommand` map to set this in flags.  A user may have a good reason to do this (I don't claim to know why).  If you do feel strongly that this needs to be done, then issue a warning so that we aren't silently ignoring their directive.

Reply:
TIL `DefValuePerCommand`, I'll use that. I will say that if a user did `skaffold dev --add-skaffold-labels=false`, things would certainly not work as expected, but I suppose that's their choice :)